code/api/src/modules/user/model.js

```diff
@@ -14,6 +14,18 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
```

Comment:
Added these new columns into our model,  we will still have to adjust our mutations and our queries in future user stories.